website/content/partials/telemetry-metrics/vault/audit/log_request_failure.mdx

```diff
@@ -2,16 +2,19 @@
 
 | Metric type | Value  | Description                                                                               |
 |-------------|--------|-------------------------------------------------------------------------------------------|
-| gauge       | number | Average (mean) number of audit log request failures across all devices during time period |
+| counter     | number | The number of audit log request failures across all devices                               |
 
 The number of request failures is a **crucial metric**.
 
-A non-zero value for `vault.audit.log_request_failure` indicates that all
-the configured audit devices failed to log a request (or response). If Vault cannot
-properly audit a request, or the response to a request, the original request
-will fail.
+When using Prometheus sink use `rate` or `irate` to convert this into the number of failures
+over a specific time period.
 
-The `mean` value for this metric should be monitored, not the `count` which could be misleading.
+When using Vault's built-in `/metrics` output format, counters are reported aggregated over 
+the metrics interval which defaults to 10 seconds. The `count` is the total number of errors that occured 
```

Comment:
This is incorrect -- I will fix it shortly.

In this case because of an [unfortunate workaround](https://github.com/hashicorp/vault/pull/3937) for [metrics not being registered properly](https://github.com/hashicorp/vault/pull/27966), the count is actually increased on both errors _and_ failures while the `mean` value is the actual mean of the samples... I'll fix that.

Reply:
Fixed in dc23fd